.github/workflows/docker_arm64.yaml

```diff
@@ -0,0 +1,32 @@
+name: DockerImage 
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/docker_arm64.yaml'
+      - 'Dockerfile'
+  push:
+    branches:
+      - "main"
+
+
+jobs:
```

Comment:
Let's use `cloud-orchestrator` directly in this context rather than `cloud-orchestration`. `cloud-orchestration` is the umbrella term for all the tools: the cloud orchestrator, cvdr and host orchestrator.